test/sql/function/string/path_join.test

```diff
@@ -0,0 +1,97 @@
+# name: test/sql/function/string/path_join.test
```

Comment:
Since we are dealing with strings, it might be good to also test with very large strings due to the german-string representation (I think you are using it correctly btw, but always good to test :) )